parsers/rust-toml.zsh

```diff
@@ -0,0 +1,15 @@
+setup() {
+	cd scripts
+	cargo build --release
+}
+
+typeset -A info=(
+	lang    'Rust'
+	toml    '1.0'
+	site    'https://github.com/toml-rs/toml'
+	src     '' # We use the Cargo.toml for now 'git@github.com:toml-rs/toml.git'
+	version '0.8.1'
+	decoder 'scripts/target/release/toml-decoder'
+	encoder 'scripts/target/release/toml-encoder'
+	perf    'scripts/target/release/rust-toml-perf'
+)
```

Comment:
Do you plan to add scripts for `toml_edit` and `basic-toml`?

Reply:
Yeah, I was working on it :-) I had to rename the binaries as they kept overriding `toml-decoder`.

Also basic-toml performance tests don't complete because it errors out; the 5M file is just generated from all the test files in toml-test, and there are 20 failing tests, so...